# There is something wrong in the Makefile such that mupdf-x11 
# does not get installed
--- Makefile	2017-04-05 07:02:21.000000000 -0400
+++ Makefile	2017-08-24 23:53:11.000000000 -0400
@@ -342,7 +342,6 @@
 $(MUJSTEST_EXE) : $(MUJSTEST_OBJ) $(MUPDF_LIB) $(THIRD_LIB)
 	$(LINK_CMD)
 
-ifeq "$(HAVE_X11)" "yes"
 MUVIEW_X11_EXE := $(OUT)/mupdf-x11
 MUVIEW_X11_OBJ := $(addprefix $(OUT)/platform/x11/, x11_main.o x11_image.o pdfapp.o)
 $(MUVIEW_X11_OBJ) : $(FITZ_HDR) $(PDF_HDR)
@@ -356,7 +355,6 @@
 $(MUVIEW_X11_CURL_EXE) : $(MUVIEW_X11_CURL_OBJ) $(MUPDF_LIB) $(THIRD_LIB) $(CURL_LIB)
 	$(LINK_CMD) $(X11_LIBS) $(CURL_LIBS) $(SYS_CURL_DEPS)
 endif
-endif
 
 ifeq "$(HAVE_GLFW)" "yes"
 MUVIEW_GLFW_EXE := $(OUT)/mupdf-gl
